### ISSUE TITLE
My E011 won't bind

### ISSUE THREAD

**Issue by LamG4** [silver13/Eachine-E011#12]
I've been troubleshooting it for quite some time now, but the issue persists. After a crash, head down, it won't bind no matters what I do. 

I've tried to bind it with a stock transmitter, and a multi protocol plugged in my Flysky transmitter.
I also flashed the regular, acro only, and factory firmware. Nothing worked.

I appreciate any help. Thanks in advance.

**Comment by Aarons14bugs** [comment]
Maybe check that the antenna is still attached?


-------- Original message --------

**Comment by brianquad** [comment]
Sounds like you may have broken your crystal oscillator. Read this page, there are a few relevant comments: https://www.rcgroups.com/forums/showthread.php?2868518-Eachine-e011-Review-Mods-and-ACRO%21%21%21%28always-in-construction%29/page49

**Comment by LamG4** [comment]
Yes the antenna still attached. It shouldn't be a problem even if I broke off the antenna because my transmitter is right next to it.

I checked the crystal oscillator and it's still attached, if it's the tube like thingy I thought it was. It'd be great if you know the part number so I can get a replacement to try on. 
I closely inspected everything and it looked like the board is fine.

**Comment by silver13** [comment]
it's not always broken on the outside, it's a 16Mhz crystal, you could use one from a tx to see if it is the crystal, it's larger but should still work.

they can be found on ebay

**Comment by LamG4** [comment]
Confirmed it was the crystal. I replaced it with a crystal from my sacrificial E010 board and now it binds just fine.
One more problem is that it kind of take some time to bind. The lights turn on dim for a good 4-5s before it goes into bind mode.
